src/components/cycle-section/learning-units-section/complete-learning-unit-toogle/CompleteLearningUnitToggleAuth.js

```diff
@@ -1,47 +1,39 @@
-import useGet from '@hooks/useGet';
 import { endpoints } from '@utils/endpoints';
-import { Skeleton } from 'primereact/skeleton';
 import style from './CompleteLearningUnitToogle.module.scss';
 import { ToggleButton } from 'primereact/togglebutton';
 
-export const CompleteLearningUnitToggleAuth = ({ unit, showSuccess }) => {
-  const completedLearningUnitEndpoint = endpoints(
-    'isLearningUnitCompleted',
-    unit.id
+export const CompleteLearningUnitToggleAuth = ({
+  learningUnit,
+  showSuccess,
+  mutate,
+}) => {
+  const completeLearningUnitEndpoint = endpoints(
+    'completeLearningUnit',
+    learningUnit.id
   );
-  const {
-    data: isCompleted,
-    isLoading,
-    isError,
-    mutate,
-  } = useGet(completedLearningUnitEndpoint);
+
   const changeHandler = async (clicked) => {
     const requestOptions = {
       method: clicked.value ? 'POST' : 'DELETE',
       headers: { 'Content-Type': 'application/json' },
     };
-    const response = await fetch(completedLearningUnitEndpoint, requestOptions);
+
+    const response = await fetch(completeLearningUnitEndpoint, requestOptions);
```

Comment:
El acuerdo era usar axios pero lo dejo a tu criterio

Reply:
We didn't use Axios at all, it would be weird to start now, even though I agree that that's what we said.